stdlib/http/src/main/ballerina/http/annotation.bal

```diff
@@ -77,8 +77,9 @@ public type Versioning record {
 # + endpoints - An array of endpoints the service would be attached to
 # + path - Path of the WebSocket service
 # + subProtocols - Negotiable sub protocol by the service
-# + idleTimeoutInSeconds - Idle timeout for the client connection. This can be triggered by putting
-#                          an `onIdleTimeout` resource in the WebSocket service.
+# + idleTimeoutInSeconds - Idle timeout for the client connection. Upon timeout, `onIdleTimeout` resource (if defined)
+#                          in the server service will be triggered. Note that this overrides the `timeoutMillis` config
+#                          in the `http:Listener`.
 # + maxFrameSize - The maximum payload size of a WebSocket frame in bytes.
 #                  If this is not set or is negative or zero the default frame size of 65536 will be used.
```

Comment:
```suggestion
#                  If this is not set or is negative or zero, the default frame size of 65536 will be used.
```
Do we need to mention the measurement unit of 65536?

Reply:
Removed it.